File: docs/API/get-scraping-jobs.code.js
let generator = client.getScrapingJobs({
	sitemap_id: 123, // optional
});
const scrapingJobs = await generator.getAllRecords();

// or iterate through all scraping jobs manually
generator = client.getScrapingJobs({
	sitemap_id: 123, // optional
});
for await (const record of await generator.fetchRecords()) {
	console.log(JSON.stringify(record));
}

File: docs/API/get-scraping-jobs.code.py
generator = client.get_scraping_jobs({
	"sitemap_id": None
})
scraping_jobs = generator.get_all_records()

# or iterate through all scraping jobs manually
generator = client.get_scraping_jobs({
	"sitemap_id": None
})
for record in generator.fetch_records():
	print(record)
